blog.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Adopt A Dog - Blog</title>
+  </head>
+
+  <body style="background-color:powderblue;">
+
+<!-- This is the header on the page - in two sections for styling later.  -->
+    <header>
+      <article id="left-header">
+        <a href="index.html"><img src="images/adoptadog-logo.svg" alt="Logo for Adopt A Dog with silouette of dog."></a>
+        <img src="images/header-image.jpg" alt="Background image of brown dog.">
+        <h2 class="header-text">Seattle's premiere dog adoption agency</h2>
+      </article>
+
+      <article id="right-header">
+        <a href="checkout.html"><img src="images/cart-icon.svg" alt="Icon of a shopping cart.">$1234.56</a>
+        <p class="checkout"><a href="checkout.html">Checkout</a></p>
+      </article>
+    </header>
+
+<!--This is the navigation - current page has id of active for styling in navigation later-->
+    <ul>
+      <li class="navigation"><a href="index.html">Home /</a></li>
+      <li class="navigation"><a href="dogs.html">Dogs /</a></li>
+      <li class="navigation"><a id="active" href="blog.html">Blog</a></li>
+    </ul>
+
+<!-- Section for blog posts -->
+    <section id="blog-content" class="grid-container-blog">
+      <h1>Adoptadog Blog</h1>
+
+      <article class="travel">
+        <img class="travel-img" src="images/blog-1.jpg" alt="Image of woman and her dog looking at the Grand Canyon.">
+        <h2 class="travel-head">Traveling With Your Dog</h2>
+        <p class="travel-desc">Doggo ipsum lotsa pats mlem heckin puggo, shoob woofer long bois what a nice floof, stop it fren shooberino. doggorino sub woofer h*ck. What a nice floof pupper extremely cuuuuuute, doggorino. I am bekom fat adorable doggo ur givin me a spook stop it fren wow such tempt noodle horse, blop doggorino you are doin me a concern super chub vvv, fat boi such treat extremely cuuuuuute bork. Yapper sub woofer h*ck yapper heckin smol borking doggo with a long snoot for pats, adorable doggo shooberino boof aqua doggo. I am bekom fat smol you are doing me a frighten very jealous pupper wow such tempt stop it fren, vvv shooberino smol. </p>
+      </article>
+
+      <article class="walk">
```

Comment:
I love how these are laid out. I'd use the same class for each article, it'll make for easier styling.

Reply:
So you mean for each of the separate articles I have on the blog.html page to just use one class that will be the same for them all?

Reply:
Yeah. If i was laying out a blog container it would look like (forgive bad intending)

<div class="blog-posts">
  <div class="blog-post></div>
  <div class="blog-post"></div>
</div>

something like that